pom.xml

```diff
@@ -217,16 +220,23 @@
             <version>1.2.3</version>
             <optional>true</optional>
         </dependency>
+        <!-- TEST -->
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <version>4.13.1</version>
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.mockito</groupId>
-            <artifactId>mockito-all</artifactId>
-            <version>1.9.5</version>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-junit4</artifactId>
+            <version>${powermock.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito</artifactId>
+            <version>${powermock.version}</version>
             <scope>test</scope>
```

Comment:
Good unit tests should not rely on hacks like this

Reply:
I mean it's perhaps not ideal, but the alternative is to have worse encapsulation it seems, leaving a lot package private and adding way more constructors